src/Error/FormattedError.php

```diff
@@ -299,7 +300,12 @@ public static function printVar($var): string
         }
 
         if (\is_object($var)) {
-            return 'instance of ' . \get_class($var) . ($var instanceof \Countable ? '(' . \count($var) . ')' : '');
+            // Calling `count` on instances of `PHPUnit\Framework\Test` triggers an unintended side effect - see https://github.com/sebastianbergmann/phpunit/issues/5866#issuecomment-2172429263
+            $count = ! $var instanceof Test && $var instanceof \Countable
```

Comment:
How does this behave when PHPUnit is not installed?

Reply:
I've tested it and it seems to be just k https://3v4l.org/FFKFf